docs/utilities/idempotency.md

```diff
@@ -2,6 +2,10 @@
 title: Idempotency
 description: Utility
 ---
+???+ warning
+	**This utility is currently in developer preview** and is intended strictly for feedback and testing purposes **and not for production workloads**. The version and all future versions tagged with the `-preview` suffix should be treated as not stable. Until this utility is [General Availability](https://github.com/awslabs/aws-lambda-powertools-dotnet/milestone/3) we may introduce significant breaking changes and improvements in response to customers feedback.
```

Comment:
Update URL to https://github.com/aws-powertools/powertools-lambda-dotnet/milestone/3